src/platform-includes/session-replay/install-session-replay/javascript.mdx

```diff
@@ -0,0 +1,28 @@
+Install the Replay package with NPM or your favourite package manager. Alternatively, you can load the Replay integration via a CDN bundle.
+
+```bash {tabTitle: ESM}
+# Using yarn
+yarn add @sentry/browser @sentry/replay
+
+# Using npm
+npm install --save @sentry/browser @sentry/replay
+```
+
+```html {tabTitle: CDN}
+<!--
+Note that the Replay bundle only contains the Replay integration and not the
+entire Sentry SDK. You have to add it in addition to the Sentry Browser SDK bundle:
+-->
+
+<script
+  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
+  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
```

Comment:
drive by review: we should make this the browser + performance bundle so it's easy for folks to use performance monitoring if they want to in the future.